### DISCUSSION TITLE
command palette: usability improvements

### DISCUSSION THREAD

**Discussion by annanay25** [grafana/grafana#50916] [Dashboards, Panels & Visualizations]
First off, thanks for the excellent feature! 🙌 

I had a couple of enhancement suggestions for the command palette and I wanted to dunk them in here for consideration:

1. Typing explore in the palette does not highlight the option to navigate to the explore page. Not sure if this was intended but it feels like a natural action to me:
![image](https://user-images.githubusercontent.com/10982987/174013145-65e655c2-7f6a-469a-aaa5-9ad6db36a7f5.png)

2. The "Navigation" item at the top of this list is different from the rest of the items in that navigation is where I currently am. I wonder if there is a better position on the palette to indicate the current position vs proposed options (this was confusing to me when I first saw it and then I figured it out):
![image](https://user-images.githubusercontent.com/10982987/174013111-c1933eb5-da37-4502-81dc-88ba6fe1cbe4.png)

Thanks!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
